easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -790,9 +790,12 @@ def all_dependencies(self):
 
         return self._all_dependencies
 
-    def dump(self, fp):
+    def dump(self, fp, always_overwrite=True, backup=False):
         """
         Dump this easyconfig to file, with the given filename.
+
+        :param overwrite: overwrite existing file at specified location without use of --force
```

Comment:
@mboisson Please also fix the '`overwrite`' in the docstring to `'always_overwrite'`

Reply:
done